client/dashboard/sites/overview-card/card.stories.tsx

```diff
@@ -3,7 +3,7 @@ import { people, seen, wordpress, backup, starEmpty, comment, envelope } from '@
 import OverviewCard, { OverviewCardProgressBar } from './';
 
 const meta = {
-	title: 'Dashboard/OverviewCard',
+	title: 'client/dashboard/OverviewCard',
```

Comment:
This is listed in the main Storybook (not `@automattic/components`), but I'm moving it because it was somewhat misplaced.